map64.go

```diff
@@ -290,6 +334,22 @@ func forEach64[K IntKey, V any](pairs []pair[K, V], f func(k K, v V) bool) {
 	}
 }
 
+func forEach64Keys[K IntKey, V any](pairs []pair[K, V], f func(k K) bool) {
+	for _, p := range pairs {
+		if p.K != K(0) && !f(p.K) {
+			return
+		}
+	}
+}
+
+func forEach64Values[K IntKey, V any](pairs []pair[K, V], f func(v V) bool) {
+	for _, p := range pairs {
+		if p.K != K(0) && !f(p.V) {
+			return
+		}
+	}
+}
+
```

Comment:
Since these 2 functions just have a single call site I would prefer if we just fold them into Keys() and Values()

Reply:
Thanks for the suggestion, done!